### PR TITLE
RavenDB-17024 Added support for arm64 docker image in build scripts

### PR DESCRIPTION
**Pull request by poissoncorp**
https://issues.hibernatingrhinos.com/issue/RavenDB-17024

### CODE REVIEW

File: docker/common.ps1 [hunk 1]
```diff
@@ -47,6 +47,15 @@ function GetUbuntuImageTags($repo, $version, $arch) {
             )
             break;
         }
+        "arm64v8" {
+            return @(
+                "$($repo):ubuntu-arm64v8-latest",
```
Comment (gregolsky) [inline docker/common.ps1]: 4.2 has different tag set. See line 38 and 51

File: docker/ravendb-ubuntu/Dockerfile.arm64v8 [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/runtime-deps:5.0-focal-arm64v8
```
Comment (gregolsky) [inline docker/ravendb-ubuntu/Dockerfile.arm64v8]: We're not using focal for 4.2 - see other docker files

File: docker/common.ps1 [hunk 3]
```diff
@@ -47,6 +47,15 @@ function GetUbuntuImageTags($repo, $version, $arch) {
...
+                "$($repo):ubuntu-arm64v8-latest-lts",
+                "$($repo):5.2-ubuntu-arm64v8-latest",
```
Comment (gregolsky) [inline docker/common.ps1]: => 4.2

**Review by gregolsky — CHANGES_REQUESTED** [review]
Please adjust based on my comments.

File: docker/common.ps1 (hunk 3 above)
Comment (poissoncorp) [reply docker/common.ps1]: done

File: docker/ravendb-ubuntu/Dockerfile.arm64v8 (hunk 2 above)
Comment (poissoncorp) [reply docker/ravendb-ubuntu/Dockerfile.arm64v8]: done

File: docker/common.ps1 (hunk 1 above)
Comment (poissoncorp) [reply docker/common.ps1]: done

**Review by gregolsky — APPROVED** [review]

**Pull request merged**